docker-compose/kitchen-sink/docker-compose.yml

```diff
@@ -377,6 +377,9 @@ services:
       zookeeperServers: zk1:2181,zk2:2181,zk3:2181
       configurationStoreServers: zk1:2181,zk2:2181,zk3:2181
       pulsar.zookeeper-uri: zk1:2181,zk2:2181,zk3:2181
+      SQL_PREFIX_pulsar.broker-service-url: http://proxy1:8080
+      SQL_PREFIX_pulsar.web-service-url: http://proxy1:8080
+      SQL_PREFIX_pulsar.zookeeper-uri: zk1:2181
```

Comment:
The prefix `SQL_PREFIX_` is not needed, it's only be used to add a new configuration to the config file.